spec/steps/execute.fmf

```diff
@@ -11,29 +11,35 @@ description: |
     in the following format::
 
         /test/one:
-            result: TEST_RESULT
+            result: OUTCOME
             log: PATH
 
         /test/two:
-            result: TEST_RESULT
+            result: OUTCOME
             log: PATH
 
-    Where ``TEST_RESULT`` is the outcome of the test. It can have
-    these values:
+    Where ``OUTCOME`` is the result of the test execution. It can
+    have the following values:
 
-        * info - test finished and produced only information
-          message
-        * passed - test finished and passed
-        * failed - test finished and failed
-        * error - a problem encountered during test execution,
-          treated as failed
+        pass
+            Test execution successfully finished and passed.
+        info
+            Test finished but only produced an informational
+            message. Represents a soft pass, automation should
+            treat this as a passed test and flag for human review.
+        warn
```

Comment:
as discussed in previous PR, all comments apply also here ...

Reply:
I will rebase once we merge...